tests/unit/views/notifications/test_project_crew_notification.py

```diff
@@ -0,0 +1,403 @@
+"""Test template strings in project crew membership notifications."""
+# pylint: disable=too-many-arguments
+
+from pytest_bdd import given, parsers, scenarios, then, when
+
+from funnel import models
+from funnel.models.membership_mixin import MEMBERSHIP_RECORD_TYPE
+
+scenarios('project_crew_notification.feature')
+
+
+@given(
+    "Vetinari is an owner of the Ankh-Morpork organization",
+)
+def given_vetinari_owner_org(
+    user_vetinari,
+    project_expo2010,
+):
+    assert 'editor' in project_expo2010.roles_for(user_vetinari)
+
+
+@given("Vetinari is an editor and promoter of the Ankh-Morpork 2010 project")
+def given_vetinari_editor_promoter_project(
+    db_session,
+    user_vetinari,
+    project_expo2010,
+):
+    user_vetinari.is_promoter = True
+    db_session.add(user_vetinari)
+    db_session.commit()
```

Comment:
These three lines can be removed. Since it's a `Given`, we're not creating fixtures, we're only confirming the existing fixture is okay. The following two `assert` statements are sufficient.